calculus.py

```diff
@@ -1,8 +1,105 @@
 """
-calculus.py
+calculus.py module
+
+This module provides implementations of the bisection root-finding algorithm 
+in three ways: using a SciPy wrapper, a pure Python implementation, 
+and a C/C++ implementation via ctypes.
```

Comment:
The module doctoring could be modified to include something more general as this module would also contain other algorithms.